### PR TITLE
Add parent object information to resolve functionality

### PR DESCRIPTION
**Pull request by dngroth**

### CODE REVIEW

File: ServerCommunication/ResolveObject.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Messerli.ServerCommunication
+{
+    public class ResolveObject
```
Comment (janhohenheim) [inline ServerCommunication/ResolveObject.cs]: How about `ResolvedObject` with a `d`? Or maybe `ObjectToResolve`?

**Review by janhohenheim — CHANGES_REQUESTED** [review]

File: ServerCommunication/ResolveObject.cs (hunk 1 above)
Comment (dngroth) [reply ServerCommunication/ResolveObject.cs]: ObjectToResolve it will be tomorrow

Comment (dngroth) [reply ServerCommunication/ResolveObject.cs]: Addressed in https://github.com/messerli-informatik-ag/server-communication/pull/3/commits/f5c3d91f0d115f09bfbd58baad29b0cb02e16f27

**Comment by janhohenheim** [comment]
Since https://github.com/messerli-informatik-ag/linq-to-rest/pull/10 was merged without these changes, can we delete this branch?

**Comment by dngroth** [comment]
Since messerli-informatik-ag/linq-to-rest#10 was merged without these changes, we can delete this branch

**Pull request closed without merging**